plutus-metatheory/src/Algorithmic/Reduction.lagda

```diff
@@ -217,6 +277,12 @@ lemSC—→V (ξ-·₂ v p) with lemSC—→V p
 lemSC—→V (ξ-·⋆ p) with lemSC—→V p
 ... | B ,, E ,, L ,, L' ,, refl ,, refl ,, q =
   B ,, E ·⋆ _ / refl ,, L ,, L' ,, refl ,, refl ,, q
+lemSC—→V (ξ-constr i {Tss} refl tidx vs cs q' p)  with lemSC—→V p
+... | B ,, E ,, L ,, L' ,, refl ,, refl ,, p' = B ,, constr i Tss refl { tidx } vs cs E ,, L ,, L' ,, 
+    constr-cong' (trans (sym (lem-≣-<>> tidx)) refl) q' (≡-subst-removable (IList ( ∅ ⊢_)) q' ((trans (sym (lem-≣-<>> tidx)) refl)) _) ,, 
+    constr-cong' (trans (sym (lem-≣-<>> tidx)) refl) q' (≡-subst-removable (IList ( ∅ ⊢_)) q' ((trans (sym (lem-≣-<>> tidx)) refl)) _) ,, p'
```

Comment:
I was suspicious about the apparently duplicated code in these two lines, but I vaguely see what's going on now.

Reply:
They look the same but the implicit arguments and the `_` are different.